plugins/dicom_viewer/server/__init__.py

```diff
@@ -65,12 +65,22 @@ def coerce(x):
         return None
 
 
-def process_file(f):
+def parse_file(f):
```

Comment:
Any reason to change this name? It's a public symbol, so this is technically a compatibility break.